### ISSUE TITLE
does not work with multiple icons

### ISSUE THREAD

**Issue by gpiwowarski-alison** [ejci/favico.js#130]
Hi, the setup on our website is as follows:

link rel="apple-touch-icon" sizes="180x180" href="apple-touch-icon.png"
link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png"
link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png"
link rel="manifest" href="manifest.json"
link rel="mask-icon" href="safari-pinned-tab.svg" color="#5bbad5"
meta name="theme-color" content="#ffffff"

The plugin only works with single icon file :(

**Comment by craigdanj** [comment]
Yep I've run into the same issue. I need a solution to this too. Do let me know if there is a way.
